utils/query_builder.py

```diff
@@ -109,6 +117,18 @@ def build(self):
         )
         return self
 
+    def build_with_order_by(self):
```

Comment:
In this method you do the same as in the build method, it would not be necessary to add it since in this way we would have a build method for each combination when the general build method already does this.
Or did the build method generate an error?